flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogStateBackendMigrationTest.java

```diff
@@ -24,28 +24,26 @@
 import org.apache.flink.runtime.state.StateBackendMigrationTestBase;
 import org.apache.flink.runtime.state.hashmap.HashMapStateBackend;
 import org.apache.flink.runtime.state.storage.JobManagerCheckpointStorage;
-
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.apache.flink.testutils.junit.extensions.parameterized.Parameter;
+import org.apache.flink.testutils.junit.extensions.parameterized.Parameters;
 
 import java.util.Arrays;
-import java.util.List;
+import java.util.Collection;
 import java.util.function.Supplier;
 
 /** Tests for the partitioned state part of {@link ChangelogStateBackend}. */
-@RunWith(Parameterized.class)
 public class ChangelogStateBackendMigrationTest
```

Comment:
```suggestion
class ChangelogStateBackendMigrationTest
```
could be package private

Reply:
`@ExtendWith(ParameterizedTestExtension.class)` cannot use package private